<launch>
 <arg name="sensorOffsetX" default="0"/>
<arg name="sensorOffsetY" default="0"/>
<node pkg="robot_simulator" type="scan_reg_sim" name="scan_reg_sim" output="screen" required="true">
    <param name="sensorOffsetX" value="$(arg sensorOffsetX)" />
    <param name="sensorOffsetY" value="$(arg sensorOffsetY)" />
</node>

</launch>